src/main/java/redis/clients/jedis/PipelineBase.java

```diff
@@ -882,7 +882,7 @@
 
   @Override
   public Response<Long> zcount(byte[] key, double min, double max) {
-    getClient(key).zcount(key, toByteArray(min), toByteArray(max));
```

Comment:
toByteArray() doesn't take care about "-inf" and "+inf" so it may be a bug around sorted set with Pipeline.